src/components/UpdateChecker.tsx

```diff
@@ -0,0 +1,60 @@
+import { useInfoToast } from '@/lib/customToast';
+import { useEffect } from 'react';
+
+const UpdateChecker = () => {
+  const currentVersion = process.env.CONFIG_CURRENT_VERSION || '0.0.0';
+  const infoToast = useInfoToast();
+
+  useEffect(() => {
+    fetch('https://raw.githubusercontent.com/Cyronlee/zBoard/main/package.json')
+      .then((res) => res.json())
+      .then((data) => {
+        const latestVersion = data.version;
+        if (compareVersions(latestVersion, currentVersion)) {
+          infoToast('New update available', 'Please update.');
```

Comment:
can we show the current version & newest version in the toast, and add a link to https://github.com/Cyronlee/zBoard/releases